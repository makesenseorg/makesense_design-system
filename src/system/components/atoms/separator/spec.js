import { shallowMount } from "@vue/test-utils";
import Comp from "./Separator.vue";

describe("Separator.vue", () => {
  it("matches snapshot", () => {
    const wrapper = shallowMount(Comp, {
      propsData: {},
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
