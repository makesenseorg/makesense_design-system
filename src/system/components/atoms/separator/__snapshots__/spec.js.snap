// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator.vue matches snapshot 1`] = `
<div
  class="separator__wrapper"
>
  <svg
    class="separator"
    height="10"
    viewBox="0 0 3258 10"
    width="3258"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 10L162.9 0l162.9 10zm651.6 0L488.7 0 325.8 10zm325.8 0L814.5 0 651.6 10zm325.8 0L1140.3 0 977.4 10zm325.8 0L1466.1 0l-162.9 10zm325.8 0L1791.9 0 1629 10zm325.8 0L2117.7 0l-162.9 10zm325.8 0L2443.5 0l-162.9 10zm325.8 0L2769.3 0l-162.9 10zm325.8 0L3095.1 0l-162.9 10z"
      style="fill: var(--color-neutral-100);"
    />
  </svg>
</div>
`;
